cmd/crio/config.go

```diff
@@ -131,6 +131,13 @@ image_volumes = "{{ .ImageVolumes }}"
 insecure_registries = [
 {{ range $opt := .InsecureRegistries }}{{ printf "\t%q,\n" $opt }}{{ end }}]
 
+# registries is used to specify registries to be used when pulling an unqualified
+# image (e.g. fedora:rawhide). "docker.io" is left here as a default registry.
+# Users can remove it and use whatever default registry they need.
+registries = [
+  "docker.io",
```

Comment:
I don't know, Docker.io is still a default since w/o this string the config isn't a working config. One can do "crio config" and get a broken config since no registries are configured. How do we deal with that?